src/screens/GroupInfo/elements/ItemLink.js

```diff
@@ -40,7 +41,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: 14,
     lineHeight: 16.94,
-    color: '#000'
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with dynamic colors from a theme object is a good practice. This improves maintainability and consistency across the application.
```diff
-    color: '#000'
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->